backend/models/project.js

```diff
@@ -0,0 +1,89 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+/**
+ * Represents every TSE project ever started since the dawn of time
+ *
+ * Contains all the people who have/are working on the project as well as other
+ * information such as project timeline, relevant files, etc.
+ */
+module.exports = mongoose.model(
+  "Project",
+  new Schema({
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    client: {
+      type: String,
+      required: true,
+    },
+    phase: {
```

Comment:
Should we make this an enum since the phases are set? I think they are: Outreach, Design, Development, Post Development